tests/deps/deps.joke

```diff
@@ -0,0 +1,15 @@
+(ns deps
+  (:require [joker.os :as os]
+            [joker.string :as str]))
+
+(def lib-dir
+  (-> (os/args)
+       (second)
+       (str/split #"/")
+       (butlast)
+       (concat ["deps" "lib"])
+       ((fn [x] (apply str (interpose "/" x))))))
+
+(ns-sources
+  {"test-local.*" {:url lib-dir}
+   "test-http.*" {:url "https://s3.us-east-2.amazonaws.com/tech.public.data/"}})
```

Comment:
Tests should not depend on external urls. For now, I'd just remove this check from the test. No tests is better the unstable tests.
I wonder how much of `procLibPath` could be rewritten in Joker. Briefly looking through the code, it should be doable. Then we could mock `joker.http/send` in the tests. This is definitely out of scope for this PR.

Reply:
- http test removed for now
- good point about serving the file from local.

Reply:
I thought about implementing more in Joker and started down that path. The trouble I ran into that made me go this way instead was based on dependencies of different things being declared where the `load-lib__` code was being done. It seemed cleaner in the end to just drop down into go and do the resolution there.

If you think otherwise, I can push further on going down the path of doing more in Joker.

Reply:
Yeah, bootstrapping may be tricky here. I'll look into it later, for now Go implementation will do.